### ISSUE TITLE
Analog Dualshock controller (SCPH-1200) gives type "Guitar Hero".

### ISSUE THREAD

**Issue by mcgurk** [SukkoPera/PsxNewLib#16]
With DumpButtonsBitBang and DumpButtonsHwSpi examples my SCPH-1200 controller gives controller type "Guitar Hero" to Serial Monitor. Other than that, controller works fully with PsxNewLib, so this doesn't matter much.

**Comment by SukkoPera** [comment]
Yeah, this is a known issue, it's even [noted in the function header](https://github.com/SukkoPera/PsxNewLib/blob/master/src/PsxNewLib.h#L804). That function is a carryover from the PS2X library, but it's undocumented and based on data reported by controllers that nobody really knows how to interpret.

I think I will just remove that function in the `devel` branch, sooner or later.

There's another similar function that is much more reliable, as it will report what protocol the library is actually using to talk to the controller. Please see [getProtocol()](https://github.com/SukkoPera/PsxNewLib/blob/master/src/PsxNewLib.h#L871).

**Comment by sonik-br** [comment]
@SukkoPera I've got an original guitar to try.
It sends DPAD LEFT as always pressed and reports as dualshock.

Holding the dpad left was also a trick to use a dualshock to play the games on ps2. Hold it when booting the game so it detects it as a guitar.

I will do more tests and see if there's any other way to identify it. But checking for the dapd state might be a good fallback.

**Comment by SukkoPera** [comment]
So I guess this function really has to go! :)

**Comment by sonik-br** [comment]
Noooooooo. Please don't kill it :)

Function is really useful to detect the presence of a jogcon. We can't rely only on the reported protocol.
Jogcon reports as digital protocol and this function can identify if it's a jogcon connected in digital mode.

Some findings on the guitar hero device. Can be useful if anyone need to identify if a guitar deice is connected:

- Dpad left is always ON.
- It boots up as DS1 in digital mode. (same behavior as a DS1)
- It supports the commands to enable Analog Sticks and Buttons. After those commands, it will report as a DS1 with DS2 protocol. (not the same behavior as a DS1)

This was tested with a wireless guitar and the original Kramer Striker Wireless Receiver.
